src/server/router-protected.ts

```diff
@@ -83,74 +82,6 @@ router.get(
   }
 );
 
-// NOTE: Fix for legacy relayApi. WMONED api is archived and is not used anymore for downloads.
-router.get(
-  BffEndpoints.WMO_DOCUMENT_DOWNLOAD,
-  async (req: Request, res: Response, next: NextFunction) => {
-    const authProfileAndToken = await getAuth(req);
-    const documentResponse = await fetchDocument(
-      res.locals.requestID,
-      authProfileAndToken,
-      req.params.id
-    );
-
-    if (
-      documentResponse.status === 'ERROR' ||
-      !documentResponse.content?.data
-    ) {
-      return res.status(500).send(documentResponse);
-    }
-
-    res.type(documentResponse.content.mimetype ?? 'application/pdf');
-    res.header(
-      'Content-Disposition',
-      `attachment; filename="${documentResponse.content.title}.pdf"`
-    );
-    return res.send(documentResponse.content.data);
-  }
-);
-
-router.use(
-  BffEndpoints.API_RELAY,
-  proxy(
-    (req: Request) => {
-      let url = '';
-      switch (true) {
-        case req.path.startsWith('/decosjoin/'):
-          url = String(process.env.BFF_VERGUNNINGEN_API_BASE_URL ?? '');
-          break;
-        case req.path.startsWith('/wpi/'):
-          url = String(process.env.BFF_WPI_API_BASE_URL ?? '');
-          break;
-        case req.path.startsWith('/brp/'):
-          url = String(process.env.BFF_MKS_API_BASE_URL ?? '');
-          break;
-        case req.path.startsWith('/wmoned/'):
-          url = String(process.env.BFF_WMO_API_BASE_URL ?? '');
-          break;
```

Comment:
Deze mag als enige weg. Voor de andere 3 routes hebben we nog geen service endpoints gemaakt in de BFF dus die proxy moet intact blijven